aws-api/src/test/java/com/amplifyframework/api/aws/auth/OwnerBasedAuthTest.java

```diff
@@ -68,6 +68,9 @@
     private static final String GRAPHQL_API_WITH_COGNITO = "graphQlApi_cognito";
     private static final String GRAPHQL_API_WITH_OIDC = "graphQlApi_oidc";
 
+    private static final String GROUP_A = "http://myapp1.com/claims/groups";
+    private static final String GROUP_B = "http://myapp2.com/claims/groups";
```

Comment:
How are group A and B different? Can  you give semantic names, `PRIVILEGED_GROUP`, `NO_ACCESS_GROUP` Or `ALICE_GROUP` and `BOB_GROUP`, with a little comment about the security relationship between Alice and Bob?

Reply:
This actually doesn't get used at the moment, but snuck its way in the PR... I'll take out the unused items from the unit test.